hack/lib/util/docs.sh

```diff
@@ -0,0 +1,139 @@
+#!/bin/bash
+#
+# This library holds utility functions related to the generation
+# of manpages and docs.
+
+
+function generate_manual_pages() {
+  local dest="$1"
+  local cmdName="$2"
+  local filestore=".files_generated_${cmdName}"
+  local skipprefix="${3:-}"
+
+  # We do this in a tmpdir in case the dest has other non-autogenned files
+  # We don't want to include them in the list of gen'd files
+  local tmpdir="${OS_ROOT}/_tmp/gen_man"
```

Comment:
probably want to do
```sh
setup_tmpdir_vars "generate/docs"
```

or similar and use some of the "normal" tmpdir locations like `$ARTIFACT_DIR`